src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -489,5 +498,146 @@ public unsafe bool HasHistoryLog(ClusterOperationContext context, BlittableJsonR
                 return true;
             }
         }
+
+        public static bool TryGetLastDeleteDatabaseCommandWithoutErrors(ServerStore serverStore, string databaseName, out long index)
```

Comment:
remove the `static` & the `serverStore` 
associate the `RachisLogHistory` with an `Engine` instead (in the `Initialize`)

Reply:
In RachisLogHistory every other method you pass a context, so I did the same here.. (I passed a context instead of serverStore)..